docs/pages/parallel-async.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Parallel and asynchronous processing
+layout: default
+---
+
+Python has a good ecosystem of libraries for multiprocessing (threads and GPU
+parallelisation), as well as asynchronous processing. Here, we list those that
+we've found to be useful, particularly for research applications and previous
+ARC projects.
```

Comment:
```suggestion
parallelisation), as well as asynchronous processing.
```
This is just a repeat of the whole purpose of the package, so worth dropping?

Reply:
Less words is less words 👍